cpp/modmesh/mesh/StaticMesh.hpp

```diff
@@ -85,7 +85,7 @@ struct CellType : NumberBase<int32_t, double>
     }
 
     uint8_t id() const { return m_id; }
-    uint8_t ndim() const { return m_ndim; }
+    uint8_t ndim() const { return static_cast<int>(m_ndim); }
```

Comment:
Why cast `m_ndim` to `int` ?
Number of dimension should not be a signed integer, it should be a unsigned integer, so this static cast should not be added.